tests/hf_hydrodata/test_point.py

```diff
@@ -151,7 +151,7 @@ def test_check_inputs():
     # Parameter provided for variable not in supported list (typo).
     point.HYDRODATA = "/hydrodata"
     with pytest.raises(Exception):
-        point.check_inputs(
+        point._check_inputs(
```

Comment:
Was this test calling a non-existent function before this?

Reply:
Yes - I don't know how this got missed when I started using pylint, but it got caught this time. Originally, "check_inputs" was the name of the function but we demoted internal-only functions to start with an underscore and it seems this test slipped through. It was running okay because the test is checking that an exception is raised.

Reply:
I think she just renamed check_inputs to _check_inputs

Reply:
@amymjohnson4000 his doesn't change the name of the function just how it's called.

@amy-defnet Wow, good catch on this. I wonder if there is a way to check which exception is raised in the future?